common/test/acceptance/tests/studio/test_studio_textbooks.py

```diff
@@ -43,9 +44,22 @@ def test_new_textbook_upload(self):
         And I have uploaded a PDF textbook and save the new textbook information
         Then the "View Live" link contains a link to the textbook in the LMS
         """
-        self.textbook_page.open_add_textbook_form()
-        self.textbook_page.upload_pdf_file('textbook.pdf')
-        self.textbook_page.set_input_field_value('.edit-textbook #textbook-name-input', 'book_1')
-        self.textbook_page.set_input_field_value('.edit-textbook #chapter1-name', 'chap_1')
-        self.textbook_page.click_textbook_submit_button()
-        self.assertTrue(self.textbook_page.is_view_live_link_worked())
+        self.textbook_upload_page.upload_new_textbook()
+        self.assertTrue(self.textbook_upload_page.is_view_live_link_worked())
+
+    @attr('a11y')
+    def test_textbook_page_a11y(self):
```

Comment:
Barring the provider login page, it looks like the only other page that I have changed is the non-iframe textbook view page (the page that the iframe is embedded in). I'll just add another accessibility test to this page before switching to the iframe.